src/store/actions.ts

```diff
@@ -0,0 +1 @@
+export { signIn } from './auth';
```

Comment:
We should re-export here all the actions
```suggestion
export { authActions } from './auth';
```